processors/app/dns/add.go

```diff
@@ -39,6 +39,16 @@ func Add(envModel *models.Env, appModel *models.App, name string) error {
 		return util.ErrorAppend(err, "failed to setup server")
 	}
 
+	// issue a warning about `.dev` being unusable with Chrome
+	if name[len(name)-4:] == ".dev" {
+		tld := appModel.DisplayName()
+		if tld == "dry-run" {
+			tld = "test"
+		}
+
+		display.Warn("\nGoogle has been locking down use of the .dev TLD, and your app may not be accessible with this domain.\nTry using %s.%s instead\n", name[0:len(name)-4], tld)
```

Comment:
I'd return that as an error (assuming the caller prints the error) so it is a hard restriction. Returning early would alleviate frustrations later when the warning is ignored and a user gets another hard error from chrome.

Reply:
Thanks for adding this in, btw.